components/script/script_thread.rs

```diff
@@ -1833,6 +1833,18 @@ impl ScriptThread {
     }
 
     fn handle_webdriver_msg(&self, pipeline_id: PipelineId, msg: WebDriverScriptCommand) {
+        match msg {
+            WebDriverScriptCommand::ExecuteScript(script, reply) => {
+                let window = { self.documents.borrow().find_window(pipeline_id) };
+                return webdriver_handlers::handle_execute_script(window, script, reply);
+            },
+            WebDriverScriptCommand::ExecuteAsyncScript(script, reply) => {
+                let window = { self.documents.borrow().find_window(pipeline_id) };
+                return webdriver_handlers::handle_execute_async_script(window, script, reply);
+            },
+            _ => (),
+        }
```

Comment:
A bit odd that we're handling these messages differently than the others? Is it worth making a similar change to the other message handles? If not, we should at least comment why these messages need handled differently than the others.

Reply:
Yeah, that's true, I added a comment explaining the difference.